packages/fleather/lib/src/widgets/editor_toolbar.dart

```diff
@@ -1,12 +1,9 @@
 import 'dart:async';
 
-import 'package:fleather/src/widgets/editor.dart';
+import 'package:fleather/fleather.dart';
```

Comment:
We should avoid importing via package for internal files, can you please use relative path?